weapon-config.inc

```diff
@@ -838,6 +838,8 @@ static s_PreviousHitI[MAX_PLAYERS];
 static Float:s_DamageDoneHealth[MAX_PLAYERS];
 static Float:s_DamageDoneArmour[MAX_PLAYERS];
 static s_DelayedDeathTimer[MAX_PLAYERS] = {-1, ...};
+static bool:s_Vehicle_Alive[MAX_VEHICLES] = true;
+static s_Vehicle_Respawn_Timer[MAX_VEHICLES] = -1;
```

Comment:
The array will be initialized having the first slot as `-1` and the rest `0`.

Proper way is like this:
```
static s_Vehicle_Respawn_Timer[MAX_VEHICLES] = {-1, ...};
```